samples/excel/45-named-item/create-and-use-named-item-for-range.yaml

```diff
@@ -0,0 +1,180 @@
+id: excel-create-and-use-named-range-item
+name: Create and use named range item
+description: Create and use a named range item
+author: OfficeDev
+host: EXCEL
+api_set:
+    ExcelApi: 1.4
+script:
+    content: |-
+        $("#add-name").click(() => tryCatch(addName));
+        $("#format-named-range").click(() => tryCatch(formatRangeByName));
+        $("#setup").click(() => tryCatch(setup));
+
+        async function addName() {
+            await Excel.run(async (context) => {
+
+                const sheet = context.workbook.worksheets.getItem("Sample");
+                const headerRange = sheet.getRange("A1:E1");
+
+                sheet.names.add("ExpensesHeader", headerRange);
+
+                const namedItems = sheet.names.load();
```

Comment:
Ricky, you're the only person who's ready my book cover-to-cover!  Certainly it was only accidental that you omitted the names of the properties that you care about in the `load` statement, right?

![image](https://user-images.githubusercontent.com/2230453/28128229-e6a8f4c0-66e3-11e7-8010-102b34eb9572.png)